projects/aws/eks-a-admin-image/Makefile

```diff
@@ -6,24 +6,24 @@ HAS_LICENSES=false
 REPO_NO_CLONE=true
 IMAGE_NAMES=
 
-REPO=eks-a-snow-admin-image
+REPO=eks-a-admin-image
 REPO_OWNER=aws
 
 BUILD_TARGETS=validate
 RELEASE_TARGETS=build-raw-image
 
 PACKER_VERSION=1.8.0
 
-EKSA_VERSION?=v0.8.0
+EKSA_VERSION?=latest
 EKSA_RELEASE_MANIFEST_URL?=https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml
 
 ARTIFACTS_UPLOAD_PATH?=$(PROJECT_PATH)
 EXPORT_AMI_BUCKET?=$(ARTIFACTS_BUCKET)
-FULL_OUTPUT_DIR=$(MAKE_ROOT)/$(OUTPUT_DIR)/ami/snow
+FULL_OUTPUT_DIR?=$(MAKE_ROOT)/$(OUTPUT_DIR)
 MANIFEST_OUTPUT?=$(FULL_OUTPUT_DIR)/manifest.json
 AMI_S3_DST=$(EXPORT_AMI_BUCKET)/$(ARTIFACTS_UPLOAD_PATH)
 EXPORT_AMI_DST=$(AMI_S3_DST)/$(EKSA_VERSION)/$(GIT_HASH)/
-LATEST_AMI_S3_URL=$(AMI_S3_DST)/$(LATEST)/snow-admin.raw
+LATEST_AMI_S3_URL=$(AMI_S3_DST)/$(LATEST)/eks-a-admin.raw
```

Comment:
Is this shared to the vsphere image? I though you would have an ova, not a raw image

Reply:
No, I'm not doing anything with raw image at the moment. I just renamed it to keep name generic. I can keep it as snow if you want to keep specific naming for each format.